src/tests/wasm.cpp

```diff
@@ -2,21 +2,32 @@
 using namespace marian::bergamot;
 
 void wasm(BlockingService &service, std::shared_ptr<TranslationModel> &model) {
-  std::vector<ResponseOptions> responseOptions;
-  std::vector<std::string> texts;
+  std::vector<std::string> plainTexts = {
+      "Hello World!",                                 //
+      "The quick brown fox jumps over the lazy dog."  //
+  };
+
+  std::vector<std::string> htmlTexts = {
+      "<a href=\"#\">Hello</a> world.",                                                   //
+      "The quick brown <b id=\"fox\">fox</b> jumps over the lazy <i id=\"dog\">dog</i>."  //
+  };
 
-  // WASM always requires HTML and alignment.
-  // TODO(jerinphilip): Fix this, bring in actual tests.
-  // responseOptions.HTML = true;
-  // responseOptions.alignment = true;  // Necessary for HTML
+  std::vector<std::string> texts;
+  std::vector<ResponseOptions> options;
+  for (auto &plainText : plainTexts) {
+    texts.push_back(plainText);
+    ResponseOptions opt;
+    options.push_back(opt);
+  }
 
-  // Hide the translateMultiple operation
-  for (std::string line; std::getline(std::cin, line);) {
-    texts.emplace_back(line);
-    responseOptions.emplace_back();
+  for (auto &htmlText : htmlTexts) {
+    texts.push_back(htmlText);
+    ResponseOptions opt;
```

Comment:
@jerinphilip I believe `opt.HTML` needs to be set to `true` here because these are html texts 👍🏾

Reply:
> Check [`c5cea4b` (#346)](https://github.com/browsermt/bergamot-translator/pull/346/commits/c5cea4b21170cac793d9f9d93b35fa5687e28dfc) for a quick confirmation this feature works. In the interest of speeding up Mozilla, I'm proposing to merge this at the earliest.
> 

Oh. I see this commit is reverted.